2021-11-18 23:07:38,162 INFO    MainThread:11495 [wandb_setup.py:_flush():71] setting env: {}
2021-11-18 23:07:38,163 INFO    MainThread:11495 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-18 23:07:38,163 INFO    MainThread:11495 [wandb_init.py:_log_setup():357] Logging user logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_230738-1jchy8rj/logs/debug.log
2021-11-18 23:07:38,163 INFO    MainThread:11495 [wandb_init.py:_log_setup():358] Logging internal logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_230738-1jchy8rj/logs/debug-internal.log
2021-11-18 23:07:38,164 INFO    MainThread:11495 [wandb_init.py:init():390] calling init triggers
2021-11-18 23:07:38,164 INFO    MainThread:11495 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-18 23:07:38,165 INFO    MainThread:11495 [wandb_init.py:init():435] starting backend
2021-11-18 23:07:38,165 INFO    MainThread:11495 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-18 23:07:38,209 INFO    MainThread:11495 [backend.py:ensure_launched():199] starting backend process...
2021-11-18 23:07:38,229 INFO    MainThread:11495 [backend.py:ensure_launched():205] started backend process with pid: 11516
2021-11-18 23:07:38,230 INFO    MainThread:11495 [wandb_init.py:init():444] backend started and connected
2021-11-18 23:07:38,231 INFO    MainThread:11495 [wandb_init.py:init():503] updated telemetry
2021-11-18 23:07:38,285 INFO    MainThread:11495 [wandb_init.py:init():533] communicating current version
2021-11-18 23:07:38,577 INFO    MainThread:11495 [wandb_init.py:init():538] got version response 
2021-11-18 23:07:38,577 INFO    MainThread:11495 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-18 23:07:38,915 INFO    MainThread:11495 [wandb_init.py:init():576] starting run threads in backend
2021-11-18 23:07:39,848 INFO    MainThread:11495 [wandb_run.py:_console_start():1693] atexit reg
2021-11-18 23:07:39,848 INFO    MainThread:11495 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-18 23:07:39,849 INFO    MainThread:11495 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-18 23:07:39,850 INFO    MainThread:11495 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-18 23:07:39,850 INFO    MainThread:11495 [wandb_init.py:init():603] run started, returning control to user process
2021-11-18 23:07:39,850 INFO    MainThread:11495 [wandb_run.py:_config_callback():911] config_cb None None {'epochs': 5, 'batch_size': 5, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_size': None, 'amp': True}
2021-11-18 23:07:39,972 INFO    MainThread:11495 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-18 23:07:39,972 INFO    MainThread:11495 [wandb_run.py:_restore():1635] restore
2021-11-18 23:07:42,879 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 810
}

2021-11-18 23:07:43,118 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 810
}

2021-11-18 23:07:43,937 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 6984
}

2021-11-18 23:07:44,040 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 6984
}

2021-11-18 23:07:44,142 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1372
  total_bytes: 6984
}

2021-11-18 23:07:44,245 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}

2021-11-18 23:07:44,347 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}

2021-11-18 23:07:44,450 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}

2021-11-18 23:07:44,553 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}

2021-11-18 23:07:44,657 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}

2021-11-18 23:07:44,857 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}

2021-11-18 23:07:45,256 INFO    MainThread:11495 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6984
  total_bytes: 6984
}
local_info {
}

2021-11-18 23:07:46,315 INFO    MainThread:11495 [wandb_run.py:_append_files():2061] logging synced files
